renderer/main.js

```diff
@@ -1106,8 +1104,6 @@ function closePlayer (cb) {
   // Tell the WebTorrent process to kill the torrent-to-HTTP server
   ipcRenderer.send('wt-stop-server')
 
-  // Tell the OS we're no longer playing media, laptops allowed to sleep again
-  ipcRenderer.send('unblockPowerSave')
```

Comment:
I remember seeing this earlier (two IPCs, one for unblockPowerSave and one for onPlayerClose) and thinking it could be cleaner 👍